index.bs

```diff
@@ -1617,8 +1617,21 @@ that are returned to the caller when a new credential is created, or a new asser
 
     dictionary AuthenticatorAttestationResponseJSON {
         required Base64URLString clientDataJSON;
-        required Base64URLString attestationObject;
+        required Base64URLString authenticatorData;
+        required long long publicKeyAlgorithm;
         required sequence<DOMString> transports;
+
```

Comment:
Does this `dictionary` look too cluttered if you remove the empty line breaks?

Reply:
I can go either way. I've removed the line breaks; tell me if it's too cluttered now :)

Reply:
> The only thing that needs tweaking now is the indentation on the first few lines of comments:

Thanks. (Tabs vs spaces. Need to configure vi on this machine.)